project_wbs/models/project_project.py

```diff
@@ -0,0 +1,342 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Luxim d.o.o.
+# Copyright 2017 Matmoz d.o.o.
+# Copyright 2017 Deneroteam.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import time
+from datetime import datetime, date
+from openerp import api, fields, models, _
+
+
+class Project(models.Model):
+    _inherit = "project.project"
+    _description = "WBS element"
+    _order = "c_wbs_code"
+
+    @api.multi
+    def _get_project_analytic_wbs(self):
```

Comment:
AFAICS the recursive Name, Code, search, etc are provided by the implicit Analytic Account.
So I feel that the corresponding code should not be in the Project.
Can you confirm this?